src/eachrow.jl

```diff
@@ -9,8 +9,11 @@
 function eachrow_replace(e::Expr)
     # Traverse the syntax tree of e
     if onearg(e, :cols)
+        @warn "cols is deprecated, use \$ instead"
```

Comment:
Maybe more explicit?
```suggestion
        @warn "cols(x) is deprecated, use \$x instead"
```